composer.json

```diff
@@ -13,9 +13,13 @@
     "keywords": ["hammock", "hack", "hacklang", "library", "mock", "mocking", "test", "testing", "php"],
     "type": "library",
     "require": {
-        "hhvm/hsl": "1.4.0"
+        "hhvm/hsl": "3.30",
```

Comment:
Do we support only 3.30 or 3.30 and above? What was the result with 4.xx?

Reply:
Lol figuring out the right versions of the dependencies was actually the hardest part of this change. These are the versions that are compatible with HHVM 3.30 - I haven't really tried fully committing to 4.x, but we can try that later.